src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -133,6 +134,10 @@ public void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
         public void updateFilteredPersonList(Predicate<ReadOnlyPerson> predicate) {
             fail("This method should not be called.");
         }
+        @Override
+        public void removeTag(Tag tags) throws DuplicatePersonException, PersonNotFoundException {
```

Comment:
To increase test coverage, need to test the method you have written.
Meanwhile, all test case should be annotated with `@Test`